### ISSUE TITLE
error getter should return Error

### ISSUE THREAD

**Issue by Niels-NTG** [reststate/reststate-vuex#24]
The `/error` getter currently returns a boolean value. Instead I would prefer this to be either the `Error` object containing the relevant error message, so I can show the appropriate error message to the user. Of course I can make my own error handlers for each case to get around this limitation of the library, but it would be nice to have.

```js
    computed: {
        error() {
            // should return Error : null, not true : false
            return this.$store.getters['publishedContent/error']
        }
    },
```

**Comment by CodingItWrong** [comment]
This makes sense. A couple of questions as I approach this:

Is the error you're thinking of for a `load` operation or for a create/update/delete? The reason I ask is that I think the code currently only updates the `error` flag for `load` operations, not for create/update/delete.

I can go ahead and expose the error for a `load` operation, but to expose the error for create/update/delete I think I need to address #19 first, to expose multiple operations' states as getters.

In the meantime, create, update, and delete should all reject with the error the server returns, so you should be able to `.catch()` it to get the data.

(Gonna leave this issue open to track implementing this for both reads and for writes.)

**Comment by Niels-NTG** [comment]
I say that it should work for both read and write operations.

Alternatively you could have general error handler that you can assign to `resourceModule`

```js
    resourceModule({
        httpClient: APIClient,
        name: 'publishedContent',
        onError(error) {
           // Do some parsing, have some side effects…
           // Return the value for the publishedContent/error getter.
           return error
        }
    })
```

**Comment by CodingItWrong** [comment]
OK, I've released v0.0.10 which exposes load operation error responses in the `error` getter, and renames the boolean `error` and `loading` getters to `isError` and `isLoading`. Updating the docs shortly. Please give it a try and let me know how it works.

As I mentioned, right now due to the architecture of the library this only exposes errors for the load operations; I'll need to address #19 to get it exposing errors for write operations. I'll leave this issue open for now since write errors aren't yet exposed.
